latest/docs-ref-conceptual/get-started-with-azure-cli.md

```diff
@@ -1,442 +1,88 @@
 ---
 title: Get started with Azure CLI 2.0
-description: Get started with Azure CLI 2.0 on Linux, Mac, or Windows.
-keywords: Azure CLI 2.0, Linux, Mac, Windows, OS X, Ubuntu, Debian, CentOS, RHEL, SUSE, CoreOS, Docker, Windows, Python, PIP
-author: rloutlaw
-ms.author: routlaw
-manager: douge
-ms.date: 02/27/2017
+description: Get started with Azure CLI 2.0
+keywords: Azure CLI, Azure management, virtual machines, vms, vnet, networks, azure scripting
+author: sptramer
+ms.author: sttramer
+manager: routlaw
+ms.date: 11/23/2017
 ms.topic: article
 ms.prod: azure
 ms.technology: azure
 ms.devlang: azurecli
 ms.service: multiple
-ms.assetid: 85c418a8-6177-4833-bb8d-ff4ce2233c1a
 ---
 
 # Get started with Azure CLI 2.0
 
-The Azure CLI 2.0 is Azure's new command line experience for managing Azure resources.
-You can use it in your browser with [Azure Cloud Shell](/azure/cloud-shell/overview),
-or you can [install](install-azure-cli.md) it on macOS, Linux, and Windows and run it from the command line.
+Azure CLI 2.0 is a command line tool for managing Azure resources. You can use it through your browser
+with [Azure Cloud Shell](/azure/cloud-shell/overview), or you can [install the CLI](install-azure-cli.md) and
+run it from your local machine.
 
-Azure CLI 2.0 is optimized for managing and administering Azure resources from the command line,
-and for building automation scripts that work against the Azure Resource Manager.
-This article helps get you started using it, and teaches you the core concepts behind it.
 
-For information about the latest release, see the [release notes](release-notes-azure-cli.md).
+The CLI is designed to make administration of Azure resources easy to understand and easy to automate. This article
+will heklp you get started by discussing some of the most common tasks performed with the CLI, introduce querying and
+output formats for getting information from the CLI outputs, and provide links to tutorials that will help understanding
+of how to do certain types of deployment and querying in the Azure ecosystem.
 
-## Connect
+## Authenticating with Azure CLI
 
-The simplest way to get started is to [launch Cloud Shell](/azure/cloud-shell/quickstart).
+If you are using [Azure Cloud Shell](/azure/cloud-shell/overview), authentication is automatically performed for you. Otherwise, you need
```

Comment:
Technically the sentence that begins with "Otherwise" is not a complete sentence, so perhaps consider beginning with something like:

> If you choose to use a local command shell, you need to use...